src/documentation/docs/Sections/InterestingFactsSection.mdx

```diff
@@ -7,43 +7,48 @@ import PropsTableRow from "@/components/internal/Documentation/PropsTable/compon
 import SlotsTableRow from "@/components/internal/Documentation/SlotsTable/components/SlotsTableRow";
 
 <Meta
-  title="Interesting Facts Section"
-  subtitle="a complete interesting facts section"
+  title="InterestingFacts Section"
+  subtitle="Allows to display a text together with a background image and numerous counters."
```

Comment:
Better description: "A section with a background image displaying a title, a text, and several counters."